redbadger/website-honestly
# Fix bugs new contacts slice

### Motivation

Fix several bugs related to the new contact us slice.
- Fix #678 
- Fix #679 
- Fix #680 

### Test plan

- Check that the new Contact Us slice appears on all the case studies
- Check that the title is "We can help you"
- Check that the email contains a subject

### Pre-merge checklist
